lib/pbench/server/api/resources/server_settings.py

```diff
@@ -19,16 +19,16 @@
     Schema,
 )
 from pbench.server.database.models.audit import AuditType
-from pbench.server.database.models.server_config import (
-    ServerConfig,
-    ServerConfigBadValue,
-    ServerConfigError,
+from pbench.server.database.models.server_settings import (
+    ServerSetting,
+    ServerSettingBadValue,
+    ServerSettingError,
 )
 
 
-class ServerConfiguration(ApiBase):
+class ServerSettings(ApiBase):
```

Comment:
Seems fine to me.  What problems do they cause?  They are related, no?

I was always confused with `ServerConfiguration` being related to our `PbenchServerConfig` objects.

Reply:
Hopefully, when someone make the typo and adds or leaves off the `s` it will be obvious and quick, but this is Python where errors aren't found until runtime and often not even then (since it's interpreted and uses duck-typing).